src/components/input-field/input-field.tsx

```diff
@@ -265,6 +273,9 @@ export class InputField {
         });
 
         window.addEventListener('resize', this.layout, { passive: true });
+        if (this.autoFocus) {
+            this.mdcTextField.focus();
```

Comment:
Like this:
![image](https://user-images.githubusercontent.com/154725/117458531-ae33c080-af4a-11eb-99e6-214926455ad9.png)

Reply:
ah okey, I just removed autofocus. What this PR does now is only adding a listener that listens to focus events. And then setting focus on the mdcTextField. Maybe we could fix autoFocus in another PR, I didn't get it to work and we only need to be able to set focus on input fields in the improve search project 🙂

Reply:
Yeah, I honestly think it would be _very_ few cases, if any, where `autofocus` would actually do anything in the Lime CRM web client. There may be other consumers where it would be more useful, so adding it wouldn't hurt, but it's fine to just add the possibility to programmatically set focus for now 👍

Reply:
Opening a search-modal, for instance ACD, is one use case where autofocus would be useful.